contents/blog/workflows-alpha.md

```diff
@@ -0,0 +1,106 @@
+---
+
+I was genuinely excited to see how users would react.  
+Maybe some quick replies. Maybe some feedback.
+And… I got replies all right.  2
+But not quite the kind I expected.
+![reply to my email 1](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha3_8f3453dfbd.png)
+![reply to my email 2](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha2_dcc58991c0.png)
+![reply to my email 3](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha4_4174b14490.png)
+---
+
+So yes, my workflow worked – a little too well.  
+Luckily, only a few users experienced the spam storm, and most took it as a joke (thankfully reducing my panic level from “delete everything” to “mild existential dread”).  
+Once I stopped laughing/crying, I fixed it and learned something useful in the process.
+
+---
+
+## About the automation
+
+Here’s what my very simple test campaign looked like:
+![automation screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha5_87ccea24d4.png)
+### Trigger
+![trigger screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha6_e74b0b9d8a.png)
+This is where you choose who gets to enter your automation.  
+I chose a **Pageview** event – meaning everyone who comes to the URL that contains *workflows* will enter this flow.
+
+The issue? I forgot to limit how often someone could enter it. The default was **once every 30 minutes**.
+
+So, if someone refreshed the page a few times, the workflow happily sent them another *“test test test.”*
+![frequency screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_frequenct_3ff32a51ff.png)
+Next, you can set a conversion goal if you want to, and choose your exit condition.
+![conversion goal screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha9_8b7b92da46.png)
+---
+
+### Delay
+
+The delay was pretty simple – I set the email to get sent **four days after** the user visited the page so they’d be reminded to use it again.
+
+Here, you can also set some conditions, if you have any (which I didn’t because I didn't think they were necessary).  
+One more thing you can add to this step is what to do if this step fails. By default, the user continues to the next step.
+![delay screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha7_68e4f7ac0d.png)
+---
+
+### Email
+
+You already saw my email, but here’s an overview of the editor:
+![email editor screenshot](https://res.cloudinary.com/dmukukwp6/image/upload/q_auto,f_auto/workflows_alpha8_a0ecf84e4a.png)
+It’s a simple, visual builder for creating and testing automated emails triggered by user behavior.  
+You can personalize fields, add dynamic properties, and drag in elements like text, buttons, or images while previewing the result on desktop or mobile.
```

Comment:
Rough steer: I think this will just present better on the page if you focus the images around the builder and editor views, and describe the rest instead.